pyirf/io/gadf.py

```diff
@@ -17,7 +17,9 @@
 
 DEFAULT_HEADER = Header()
 DEFAULT_HEADER["CREATOR"] = f"pyirf v{__version__}"
-DEFAULT_HEADER["HDUDOC"] = "https://github.com/open-gamma-ray-astro/gamma-astro-data-formats"
+DEFAULT_HEADER[
```

Comment:
Is this a black change? I don't really like it. It is less readable than before I think

Reply:
You can add a `# fmt: off` comment at the end of the line to keep black from interfering here.